forms-flow-api/src/formsflow_api/models/form_bundling.py

```diff
@@ -29,3 +29,20 @@ class FormBundling(BaseModel, db.Model):
     parent_form_id = db.Column(
         db.String(50), nullable=False, comment="parent id of form inside bundle"
     )
+
+    @classmethod
+    def create_from_dict(cls, mapper_info: dict):
+        """Create new mapper between form and process."""
+        form_bundlings = []
+        for form_info in mapper_info["selected_forms"]:
+            form_bundling = cls()
+            form_bundling.rules = form_info.get("rules")
+            form_bundling.path_name = form_info.get("path")
+            form_bundling.mapper_id = form_info.get("mapperId")
+            form_bundling.form_process_mapper_id = form_info.get("formProcessMapperId")
+            form_bundling.form_order = form_info.get("formOrder")
+            form_bundling.parent_form_id = form_info.get("parent_form_id")

```

Comment:
please follow either camelCase or snake_case consitently.
Here parent_form_id is in snake case, while all others are camel

Reply:
And I think this will be in snake_case, as you are loading the data to schema first